maker/src/actor_system.rs

```diff
@@ -247,7 +247,7 @@ where
             let identity = identity.libp2p.clone();
             move || {
                 identify::listener::Actor::new(
-                    vergen_version::git_semver().to_string(),
+                    daemon::VERSION.to_string(),
```

Comment:
maybe we could add a pub method in daemon/lib.rs that just returns `version() -> String` ? this way it's more flexible and less noisy in all the code here.